packages/multi-test/src/app.rs

```diff
@@ -162,28 +251,23 @@ where
     }
 }
 
-pub struct Router<C> {
-    pub wasm: Box<dyn Wasm<C>>,
-    pub bank: Box<dyn Bank>,
+pub struct Router<ExecC, QueryC> {
+    pub(crate) wasm: Box<dyn Wasm<ExecC, QueryC>>,
```

Comment:
I would actually leave these public. But that can be a different PR when there is a need.

Reply:
Why? Any reason why user of this crate should in any point get access to internals of structure?

Reply:
Not now, I have an idea, but the need will show up in a future PR.
Out of scope here. For now pub(crate) is good.